application/backend/app/alembic/versions/2786b50eb5a4_schema.py

```diff
@@ -104,6 +106,9 @@ def upgrade() -> None:
             ["dataset_revisions.id"],
         ),
         sa.PrimaryKeyConstraint("id"),
+        sa.Index("idx_model_revisions_project", "project_id"),
+        sa.Index("idx_model_revisions_project_status", "project_id", "training_status"),
+        sa.Index("idx_model_revisions_architecture", "architecture"),
```

Comment:
```suggestion
        sa.Index("idx_model_revisions_project_status", "project_id", "training_status"),
        sa.Index("idx_model_revisions_project_architecture", "project_id", "architecture"),
```
- index (project) is redundant because it's prefix to (project, training_status)
- architectures are always queried in the context of a project, therefore (architecture) should be (project, architecture)